src/lib/Microsoft.Health.Events/Microsoft.Health.Events.csproj

```diff
@@ -39,4 +39,17 @@
     <ProjectReference Include="..\Microsoft.Health.Common\Microsoft.Health.Common.csproj" />
     <ProjectReference Include="..\Microsoft.Health.Logger\Microsoft.Health.Logging.csproj" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Not sure if we needed these files. If not you can revert this change

Reply:
Yes, I added this resource file to localize the strings that we expose to the customers, in this case I've added the custom exception messages.